packages/envied_generator/lib/src/generate_field_encrypted.dart

```diff
@@ -43,7 +84,7 @@ Iterable<Field> generateFieldsEncrypted(FieldElement field, String value) {
         (FieldBuilder fieldBuilder) => fieldBuilder
           ..static = true
           ..modifier = FieldModifier.final$
-          ..type = refer('int')
+          ..type = refer('int$nullability')
```

Comment:
The reason I have refactored this library to use `code_builder` is so that we don't have to do string interpolation like this anymore. Please refactor all of your code using `$nullability` string interpolation to something like this:

```dart
..type = TypeReference(
  (b) => b
    ..symbol = 'int'
    ..isNullable = true
)
```

Find more examples [here](https://github.com/dart-lang/code_builder/blob/master/test/specs/type_reference_test.dart#L29)

Reply:
Thanks!